libc/include/math.yaml

```
@@ -2440,6 +2440,12 @@ functions:
     arguments:
       - type: _Float16
     guard: LIBC_TYPES_HAS_FLOAT16
+  - name: sinpi
+    standards:
+      - stdc
+    return_type: double
+    arguments:
+      - type: double
   - name: sinpif16
```

Can you also add `sinpif`? It's already implemented but missing in math.yaml for some reason.

```suggestion
  - name: sinpi
    standards:
      - stdc
    return_type: double
    arguments:
      - type: double
  - name: sinpif
    standards:
      - stdc
    return_type: float
    arguments:
      - type: float
  - name: sinpif16
```